mimic/test/test_nova.py

```diff
@@ -39,6 +39,27 @@ def setUp(self):
         self.server_id = self.create_server_response_body['server']['id']
         self.nth_endpoint_public = helper.nth_endpoint_public
 
+    def test_create_server_with_manual_diskConfig(self):
+        """
+        Servers should respect the provided OS-DCF:diskConfig setting if
+        supplied.
+        """
+        create_server = request(
+            self, self.root, "POST", self.uri + '/servers',
+            json.dumps({
+                "server": {
+                    "name": self.server_name + "A",
+                    "imageRef": "test-image",
+                    "flavorRef": "test-flavor",
+                    "OS-DCF:diskConfig": "MANUAL",
+                }
+            }))
+        create_server_response = self.successResultOf(create_server)
+        response_body = self.successResultOf(
+            treq.json_content(create_server_response))
+        self.assertEqual(
+            response_body['server']['OS-DCF:diskConfig'], 'MANUAL')
```

Comment:
It would be good to also assert something about the `disk_config` attribute itself, or, failing that, the results of a subsequent `GET` request; this test could be made to pass by simply returning a different static value for the create response and *not* properly updating Mimic's internal state to reflect that server's disk configuration.